Sources_v3/Database/DTOs/MessageDTO.swift

```diff
@@ -58,6 +58,14 @@ class MessageDTO: NSManagedObject {
         return request
     }
     
+    /// Returns a fetch request for messages pending edit.
+    static func messagesPendingEditFetchRequest() -> NSFetchRequest<MessageDTO> {
```

Comment:
I'm not sure about `pendingEdit` wording. It's not the message "waiting to be edited" as the name suggests. It's messages that have been edited and they are waiting to be synced. So maybe, `pendingSync`? Same for the local state.